client/src/components/Ticketing/ticketingmanager/SubscriptionRedemption/SubscriptionsRedemptionContainer.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  TextField,
+  Autocomplete,
+  Select,
+  MenuItem,
+  InputLabel,
+  FormControl,
+} from '@mui/material';
+import React, {useEffect, useState} from 'react';
+import {
+  getSubscriptionFilterFunction,
+  getSubscriptionSortFunction,
+  Subscription,
+  useFetchSeasons,
+} from './SubscriptionRedemptionUtils';
+import {SubscriptionRedemptionContainer} from './SubscriptionRedemptionContainer';
+import {getData} from '../Event/components/ShowingUtils';
+
+interface SubscriptionsRedemptionProps {
+  token: string;
+  setPopUpProps: any;
+  showPopUp: boolean;
+}
+
+export const SubscriptionsRedemptionContainer = (
+  props: SubscriptionsRedemptionProps,
+) => {
+  const {token, showPopUp, setPopUpProps} = props;
+  const {seasons} = useFetchSeasons('?current=true');
+  const [subscriptions, setSubscriptions] = useState<Subscription[]>([]);
+  const [inputValue, setInputValue] = useState<string>('');
+  const [queries, setQueries] = useState({
+    seasons: [],
+    name: '',
+    email: '',
+  });
+  const [filters, setFilters] = useState({
+    filter: 3,
+    sort: 3,
+  });
+  const [editing, setEditing] = useState(false);
+  const [search, setSearch] = useState(false);
+
+  useEffect(() => {
+    const controller = new AbortController();
+    if (token) {
+      const queriesFormatted = queries.seasons.map(
+        (season) => `seasonid=${season.id}`,
+      );
+
+      if (queries.name) {
+        queriesFormatted.concat(
+          queries.name
+            .trim()
+            .split(' ')
+            .map((name) => `name=${name}`),
+        );
+      }
+
+      getData(
+        `${
+          process.env.REACT_APP_API_2_URL
+        }/subscription-types/subscriptions/search?${queriesFormatted.join(
+          '&',
+        )}${queries.email ? `&email=${queries.email.trim()}` : ''}`,
+        setSubscriptions,
+        controller.signal,
+        token,
+      ).catch((error) => console.error(error, 'Error Fetching Subscriptions'));
+    }
+    return () => controller.abort();
+  }, [search]);
+
+  return (
+    <main className='grid grid-cols-12 gap-2 min-[2000px]:w-[70%]'>
+      <header className='col-span-12'>
+        <h1 className='font-bold text-5xl bg-clip-text text-transparent bg-gradient-to-r from-blue-300 to-indigo-500 mb-14'>
+          Subscriptions
+        </h1>
+      </header>
+      <div className='col-span-12 mb-4 min-[1200px]:mb-0 min-[1200px]:col-span-4 min-[1440px]:col-span-3'>
```

Comment:
`min-[1440px]` should be covered by the `lg` shorthand

Reply:
Updated